docs/guides/securing-apps/authz-client.adoc

```diff
@@ -168,4 +168,80 @@ for (Permission granted : requestingPartyToken.getPermissions()) {
 }
 ----
 
+[[_client_authentication_adapter]]
+== Client authentication
+
+When an authorization client needs to send a backchannel request, it needs to authenticate against the {project_name} server. By default, there are three ways to authenticate the client: client ID and client secret, client authentication with signed JWT, or client authentication with signed JWT using client secret.
+
+=== Client ID and Client Secret
+
+This is the traditional method described in the OAuth2 specification. The client has a secret, which needs to be known to both the client and the {project_name} server.
+You can generate the secret for a particular client in the {project_name} Admin Console, and then paste this secret into the `keycloak.json` file on the application side:
+
+
+[source,json]
+----
+"credentials": {
+    "secret": "19666a4f-32dd-4049-b082-684c74115f28"
+}
+----
+
+=== Client authentication with Signed JWT
+
+This is based on the https://datatracker.ietf.org/doc/html/rfc7523[RFC7523] specification. It works this way:
+
+* The client must have the private key and certificate. For authorization client this is available through the traditional `keystore` file, which is either available on the client application's classpath or somewhere on the file system.
+
+* During authentication, the client generates a JWT token and signs it with its private key and sends it to {project_name} in
+the particular request in the `client_assertion` parameter.
+
+* {project_name} must have the public key or certificate of the client so that it can verify the signature on JWT. In {project_name} you need to configure client credentials for your client. First you need to choose `Signed JWT` as the method of authenticating your client in the tab `Credentials` in the Admin Console.
```

Comment:
```suggestion
* {project_name} must have the public key or certificate of the client so that it can verify the signature on JWT. In {project_name}, you configure client credentials for your client. First, you choose `Signed JWT` as the method of authenticating your client in the tab `Credentials` in the Admin Console.
```